src/main/resources/application.properties

```diff
@@ -1 +1,14 @@
 
+# ===============================
+# = JPA / HIBERNATE
+# ===============================
+#spring.jpa.show-sql = true
+spring.jpa.hibernate.ddl-auto=update
+spring.jpa.hibernate.naming-strategy = org.hibernate.cfg.ImprovedNamingStrategy
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
+#spring.jpa.open-in-view=false
+
+spring.datasource.url: jdbc:mysql://localhost:3306/sonot?allowPublicKeyRetrieval=true&useSSL=false&autoReconnect=true&characterEncoding=UTF-8&connectionCollation=utf8mb4_unicode_ci&characterSetResults=UTF-8&useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
+spring.datasource.username=root
```

Comment:
Another user with lesser privileges is better.

Reply:
of course but it was just for texting the models